packages/ui/src/components/collectibles/native/collectible-html.native.tsx

```diff
@@ -0,0 +1,15 @@
+import { WebView } from 'react-native-webview';
+
+import { CollectibleCardLayout } from './collectible-card-layout.native';
+
+interface CollectibleHtmlProps {
+  src: string;
+}
+
+export function CollectibleHtml({ src }: CollectibleHtmlProps) {
+  return (
+    <CollectibleCardLayout>
+      <WebView source={{ uri: src }} width={200} height={200} style={{ overflow: 'hidden' }} />
```

Comment:
I need to do a bit more work here as I still see some scroll bars

https://github.com/user-attachments/assets/eee72595-9e99-46d1-8359-94342ac8d0cb